common/persistence/sql/sqlplugin/postgresql/visibility.go

```diff
@@ -304,41 +135,92 @@ func (pdb *db) GetFromVisibility(
 	filter sqlplugin.VisibilityGetFilter,
 ) (*sqlplugin.VisibilityRow, error) {
 	var row sqlplugin.VisibilityRow
-	err := pdb.conn.GetContext(ctx,
-		&row,
-		templateGetWorkflowExecution,
-		filter.NamespaceID,
-		filter.RunID,
-	)
+	stmt, err := pdb.conn.PrepareNamedContext(ctx, templateGetWorkflowExecution_v12)
+	if err != nil {
+		return nil, err
+	}
+	err = stmt.GetContext(ctx, &row, filter)
+	if err != nil {
+		return nil, err
+	}
+	err = pdb.processRowFromDB(&row)
 	if err != nil {
 		return nil, err
 	}
-	pdb.processRowFromDB(&row)
 	return &row, nil
 }
 
 func (pdb *db) CountFromVisibility(
 	ctx context.Context,
 	filter sqlplugin.VisibilitySelectFilter,
 ) (int64, error) {
-	return 0, store.OperationNotSupportedErr
+	var count int64
```

Comment:
I think it would be good to reflect these changes a little more in the description of the PR. Like "Remove support for standard visibility. Previously unsupported postgres operations that were supported by postgres v12 are now supported because postgres v12 is the default." Or some acknowledgement that the deprecation of standard visibility includes changes such as these.

Reply:
These are known features of advanced visibility.